### PR TITLE
first terrafrm script to create rg and storage account

### PR DESCRIPTION
**Pull request by Swapna1603**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+provider "azurerm" {
+  features {}
+}
+
+# Create a resource group
+resource "azurerm_resource_group" "rg" {
+  name     = "my-resource-group"
```
Comment (techslateramu) [inline main.tf]: Can you change it to your name - say swapna01

Comment (Swapna1603) [reply main.tf]: Hi...Changes made.

**Review by techslateramu — APPROVED** [review]
Appproved !!

File: main.tf (hunk 1 above)
Comment (Hemalika) [reply main.tf]: approved!!
you can merge the code

**Comment by Hemalika** [comment]
approved!!
you can merge the code

**Comment by harshini6395** [comment]
Swapna you didn't follow the structure like allinone repo. So, redo the exercise then raise the PR

**Comment by Swapna1603** [comment]
> Swapna you didn't follow the structure like allinone repo. So, redo the exercise then raise the PR

hi Harshini, made the changes. Request you to approve

**Comment by harshini6395** [comment]
aPPROVED.

**Comment by Hemalika** [comment]
Approved

**Pull request merged**